### ISSUE TITLE
Markup differences between /examples/ and /examples-full/ causing a11y issues

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-next#1693]
If you go to https://pf4.patternfly.org/components/Nav/examples/ and look at the "horizontal nav" example, there is a dark background that comes from the `.is-dark-preview` class in the example.

```
      <Example
        heading="Horizontal Nav (only in masthead)"
        handlebars={navHorizontalListExampleRaw}
        className="is-dark-preview"
      >
```

Then we style it with this CSS in [workspace.scss](https://github.com/patternfly/patternfly-next/blob/master/src/site/workspace.scss)

```
.is-dark-preview {
  .Preview__body {
    background-color: var(--pf-global--BackgroundColor--dark-100);
  }
}
```

However, if you go to https://pf4.patternfly.org/components/Nav/examples-full/, the example (second one from the bottom) doesn't have a dark background because the markup is different between the `/examples/` and `/examples-full/` pages. That example fails a11y because we rely upon it being placed in the page header, which has a dark background.

**Comment by mcoker** [comment]
cc @seanforyou23 @dgutride - is this something we could fix in the workspace? For example, move the `.is-dark-preview` class (and any other dark theme classes that get applied) to an element that wraps just the component code, and surface that in `/examples-full/`?

**Comment by redallen** [comment]
This issue is out of date. The issues still exist for https://pf4.patternfly.org/documentation/core/components/nav/default and others. An issue can be opened to have fullscreen examples apply workspace styles in https://github.com/patternfly/patternfly-org . To see our a11y checker fail these examples, remove the `color-contrast` ignore in package.json:
```sh
patternfly-a11y --file .cache/fullscreenPages.json --prefix http://localhost:9000 --aggregate --noIncomplete --ignore color-contrast,focusable-content,scrollable-region-focusable
```
to
```sh
patternfly-a11y --file .cache/fullscreenPages.json --prefix http://localhost:9000 --aggregate --noIncomplete --ignore focusable-content,scrollable-region-focusable
```
